File: basic/Class_Member/3.py
class Cs :
    count = 0;
    def __init__(self) :
        Cs.count = Cs.count + 1
    @classmethod
    def getCount(cls) :
        # cls는 class를 가리킨다
        return cls.count

i1 = Cs()
i2 = Cs()
i3 = Cs()
print(Cs.getCount())

File: basic/Class_Member/3.rb
class Cs
  @@count = 0
  def initialize()
    # @@ : 클래스에 속한 변수
    # @ : 인스턴스에 속한 변수
    @@count = @@count + 1
  end
  def Cs.getCount()
    return @@count
  end
end

i1 = Cs.new()
i2 = Cs.new()
i3 = Cs.new()
p Cs.getCount()
